README.md

```diff
@@ -25,120 +23,168 @@ Today, the attestation model does not seek to provide information to users withi
+   </td>
   </tr>
   <tr>
-   <td><code>protected_audience_api</code></td>
-   <td>Attestations for the Protected Audience APIs. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean&gt;</code> the attestation strings the ad tech attests to.</td>
+   <td><code>protected_audience_api</code>
+   </td>
+   <td>Attestations for the Protected Audience APIs. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean></code> items that indicate the attestation strings the ad tech attests to.
+   </td>
   </tr>
   <tr>
-   <td><code>shared_storage_api</code></td>
-   <td>Attestations for the Shared Storage API. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean&gt;</code> the attestation strings the ad tech attests to.  </td>
+   <td><code>shared_storage_api</code>
+   </td>
+   <td>Attestations for the Shared Storage API. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean></code> items, that indicate the attestation strings the ad tech attests to.
+   </td>
   </tr>
   <tr>
-   <td><code>private_aggregation_api</code></td>
-   <td>Attestations for the Private Aggregation API. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean&gt;</code> the attestation strings the ad tech attests to. </td>
+   <td><code>private_aggregation_api</code>
+   </td>
+   <td>Attestations for the Private Aggregation API. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean></code> items, that indicate the attestation strings the ad tech attests to. 
+   </td>
   </tr>
 </table>
 
 The attestation file is verified for consistency regularly to ensure the language included within the file remains unchanged and on the ad tech's server in the correct location. It's important to ensure that developers do not maintain multiple versions of the attestation file. In the future, it may be helpful for browsers or devices to routinely observe attestation files so that differences from expectation attestations could be highlighted to the ecosystem. 
 
-We strive to bring enhanced transparency to the use of the Privacy Sandbox relevance and measurement APIs. Therefore, besides placing the file on a server, we will generate reports that share enrollment and attestation related information for each enrolled developer. These reports will be designed to help a wide variety of audiences, including end users, governments and KOFs, gain more understanding and context around who is using these APIs. 
+We strive to bring enhanced transparency to the use of the Privacy Sandbox relevance and measurement APIs. Therefore, besides placing the file on a server, we will generate reports that share enrollment and attestation related information for each enrolled developer. These reports will be designed to help a wide variety of interested parties gain more understanding and context around who is using these APIs. 
+
+## Notes
```

Comment:
I don't think this needs a separate heading, as it generates a double separating line.